external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaWriteTask.scala

```diff
@@ -32,7 +31,7 @@ import org.apache.spark.sql.types.{BinaryType, StringType}
  * automatically trigger task aborts.
  */
 private[kafka010] class KafkaWriteTask(
-    producerConfiguration: ju.Map[String, Object],
+    producerConfiguration: ju.HashMap[String, Object],
```

Comment:
Ideally this should not have been changed. But, they are changed to HashMap, to avoid converting or casting them later.